service/restore/restoreRunner.go

```diff
@@ -6,28 +6,32 @@ import (
 
 	backupv1alpha1 "github.com/vshn/k8up/apis/backup/v1alpha1"
 	"github.com/vshn/k8up/service"
+	"github.com/vshn/k8up/service/observe"
 	"github.com/vshn/k8up/service/schedule"
+	batchv1 "k8s.io/api/batch/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-type RestoreRunner struct {
+type restoreRunner struct {
 	restore *backupv1alpha1.Restore
 	service.CommonObjects
-	config config
+	config   config
+	observer *observe.Observer
 }
 
 // NewRestoreRunner returns a new restore runner
-func NewRestoreRunner(restore *backupv1alpha1.Restore, common service.CommonObjects) *RestoreRunner {
-	return &RestoreRunner{
+func NewRestoreRunner(restore *backupv1alpha1.Restore, common service.CommonObjects, observer *observe.Observer) *restoreRunner {
```

Comment:
Why unexport `restoreRunner` when it is returned by exported `NewRestoreRunner`?

Reply:
Good catch, `NewRestoreRunner` needs to be unexported of course.